settings.py

```diff
@@ -234,7 +234,7 @@
 
 GUNICORN_PID_PATH = "/var/run/gunicorn.pid"
 
-LOG_LEVEL = logging.INFO
```

Comment:
You should add this as a value under `local_settings.py` so it doesn't get persisted to production.

Reply:
This is for debug, not meant to actually land

Reply:
ah, ok. That said, it may not be a bad thing to have this always be there in `local_settings.py`

Reply:
Actually we already do: https://github.com/whamcloud/integrated-manager-for-lustre/blob/master/tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup#L81-L85

And it's present and shows up via `python manage.py diffsettings|grep LOG`

But I'm not seeing DEBUG logging from all plugins/jobs.

Reply:
It think this only applies for the first test though. At some point this file is getting removed, I think when we re-install the manager between tests?